### PR TITLE
RD-2934 Add system tests for "Service Button" widget

### PR DESCRIPTION
**Pull request by qooban**
## Description
Added system tests for new "Service Button" widget. This PR builds on top of #1555 .

Additional changes:
* Created custom Cypress commands for changing widget configuration fields (boolean, string and searchable dropdown) to `test/cypress/support/editMode.ts`
* Created custom Cypress command for rendering widget with default configuration (`useWidgetWithDefaultConfiguration`) - it would be wise probably to reuse it in other widgets as well, it seems very common case (I can do that within this PR or create another one)
* Updated "Number of Deployments" widget spec to use new Cypress commands

## Screenshots / Videos

https://user-images.githubusercontent.com/5202105/130203745-8db81510-30b4-4ec7-8c15-1c309947c7b6.mp4

## Checklist
- [x] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires a documentation update.
- [x] I added proper labels to this PR.

## Tests
[Stage-UI-System-Test/1115](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/1115/pipeline) - PASSED

## Documentation
N/A

### CODE REVIEW

File: test/cypress/support/commands.ts [hunk 1]
```diff
@@ -206,7 +205,7 @@ const commands = {
             );
             if (disableGettingStarted) mockGettingStarted(false);
         });
-        cy.visit('/console');
+        return cy.visit('/console');
```
Comment (qooban) [inline test/cypress/support/commands.ts:208]: All changes above this line are to make TypeScript happy and not show warnings about returning `void` and using chaining functions. I think we should update all the custom cypress commands to return `Chainable` properly.
What do you think? (if we agree, I'll create a ticket for that and maybe I'll be able to handle it soon)

Comment (kubama) [reply test/cypress/support/commands.ts:208]: Yes, I agree, I can see no reason why it shouldn't be this way.

File: test/cypress/support/commands.ts [hunk 2]
```diff
@@ -319,6 +318,13 @@ const commands = {
             }
         });
     },
+    useWidgetWithDefaultConfiguration: (widgetId: string, widgetConfigurationOverrides: any = {}) =>
+        cy
+            .usePageMock(widgetId, widgetConfigurationOverrides)
+            .mockLogin()
+            // In order to load default configuration for widget
+            // open widget edit configuration modal and save without making any changes
+            .editWidgetConfiguration(widgetId, noop),
```
Comment (kubama) [inline test/cypress/support/commands.ts:327]: We have RD-1820 to address the issue differently, maybe worth to reference it in the comment here?

**Review by kubama — APPROVED** [review]

File: test/cypress/support/commands.ts (hunk 2 above)
Comment (qooban) [reply test/cypress/support/commands.ts:327]: I wasn't aware of that issue. Definitely, I'm adding a reference to that issue in the comment.

File: test/cypress/support/commands.ts (hunk 1 above)
Comment (qooban) [reply test/cypress/support/commands.ts:208]: I decided to do better research first, to see if that should be applied for all custom commands. Added a point to UI weekly meeting about it.

**Review by kubama — APPROVED** [review]

**Pull request merged**